### DISCUSSION TITLE
connection

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#26504] [General]

**Comment by potiuk** [comment]
You should percent encode your username. See https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html and when you scroll down you will see exact instructions on how you can connect proper URI

**Reply by potiuk** [reply]
Read the docs I posted above. If you read it, yo will see that it explores all options on how you can manage connections.
If it does not work, then it is likely a problem with how you are tryiung to define it.

So far you have not described how you are trying to setup, and no-one here can magically guess it. So if you have a problem with some concrete way of how you are trying to set it up - i suggest you describe all the details and explain all the errors and logs (screenshots) and show the investigation and ways you tried and it did not work - this might actually help others to help you in (likely) correcting the mistake,

But if you just write "I tried and it does not work" without any explanation and detail, you do not give others a chance to help you.
